src/main/steps/la-portal/task-list/template.njk

```diff
@@ -0,0 +1,64 @@
+{% extends "common/page.njk" %}
```

Comment:
Now I realised, application > task-list template can be used here.. it's exact copy